src/screens/TransactionConfirmationScreen.js

```diff
@@ -420,30 +418,26 @@ const TransactionConfirmationScreen = () => {
       }
     }
 
-    const web3TxnCount = await getTransactionCount(txPayload.from);
-    const maxTxnCount = Math.max(transactionCountNonce, web3TxnCount);
-    const nonce = maxTxnCount;
     const calculatedGasLimit = gas || gasLimitFromPayload || gasLimit;
-    let txPayloadLatestNonce = {
+    let txPayloadUpdated = {
```

Comment:
We can now remove redux/nonce and any references to `updateTransactionCountNonce` and `nonceClearState`